### PR TITLE
VideoPress: capital P

### PR DESCRIPTION
**Pull request by retrofox**
`VideoPress` > `Videopress`

Fixes #

#### Changes proposed in this Pull Request:

* VideoPress: capital P

#### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?

#### Testing instructions:

Build VideoPress plugin

```cli
jetpack build plugins/videopress
```

* Confirm the Admin page works as expected
* Confirm the `jetpackVideoPressInitialState` global is exposed properly.

image

### CODE REVIEW

File: projects/packages/videopress/src/class-status.php [hunk 1]
```diff
@@ -38,6 +38,6 @@ public static function is_jetpack_active() {
 	 * @return boolean
 	 */
 	public static function is_standalone_plugin_active() {
-		return class_exists( 'Jetpack_Videopress_Plugin' );
+		return class_exists( 'Jetpack_VideoPress_Plugin' );
```
Comment (leogermani) [inline projects/packages/videopress/src/class-status.php:41]: need to rename the class declaration as well and check all occurrences calling it

File: projects/packages/videopress/package.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
 	"description": "VideoPress package",
 	"homepage": "https://github.com/Automattic/jetpack/tree/HEAD/projects/packages/videopress/#readme",
 	"bugs": {
-		"url": "https://github.com/Automattic/jetpack/labels/[Package] Videopress"
+		"url": "https://github.com/Automattic/jetpack/labels/[Package] VideoPress"
```
Comment (kraftbj) [inline projects/packages/videopress/package.json]: I think the project structure linter (.github/files/lint-project-structure.sh IIRC) would need to be updated to allow mixed case. Right now, it just sees `videopress` as the folder so wants `Videopress`. @anomiex

Comment (retrofox) [reply projects/packages/videopress/package.json]:
```suggestion
		"url": "https://github.com/Automattic/jetpack/labels/[Package] Videopress"
```

**Review by oskosk — APPROVED** [review]
Tests well

File: projects/packages/videopress/package.json (hunk 2 above)
Comment (anomiex) [reply projects/packages/videopress/package.json]: Copying my reply from Slack to here to close the loop:

> You'd have to add a special case near [.github/files/lint-project-structure.sh#L153](https://github.com/Automattic/jetpack/blob/9cdc2430d8448c3ce7915e3a473b9730ccea1a14/.github/files/lint-project-structure.sh#L153) and [.github/files/lint-project-structure.sh#L306](https://github.com/Automattic/jetpack/blob/9cdc2430d8448c3ce7915e3a473b9730ccea1a14/.github/files/lint-project-structure.sh#L306) (maybe abstracting it into a function) to generate the correct URL, and also add one to the function at [projects/github-actions/repo-gardening/src/tasks/add-labels/index.js#L15](https://github.com/Automattic/jetpack/blob/9cdc2430d8448c3ce7915e3a473b9730ccea1a14/projects/github-actions/repo-gardening/src/tasks/add-labels/index.js#L15) to make it assign the correct label. Also possibly something near [projects/github-actions/repo-gardening/src/tasks/check-description/index.js#L118](https://github.com/Automattic/jetpack/blob/trunk/projects/github-actions/repo-gardening/src/tasks/check-description/index.js#L118) too. Maybe other places too.

**Pull request merged**